.github/workflows/pr.yaml

```diff
@@ -157,7 +157,7 @@ jobs:
           # Warning: the ' in build-args are fragile -- only use on LDFLAGS
```

Comment:
I guess we should remove this comment too

Reply:
as I was writing the commit message I literally saw this and went 'must fix that' and yet, here we are (done now)